### PR TITLE
fix and add back test_tutorials_pytorch.py

### PR DESCRIPTION
**Pull request by yzhang123**
Signed-off-by: Yang Zhang <yangzhang@nvidia.com>

### CODE REVIEW

**Comment by blisc** [comment]
Do we really want this unittest? It just tests some random function `loadPrepareData` but doesn't seem to cover all core or important nemo functionality.

File: tests/data/dialog_sample.txt [hunk 1]
```diff
@@ -1,1000 +1,1000 @@
...
-That's what he did to me.  He put cigarettes out on me.    Your father put cigarettes out on you?
-Your father put cigarettes out on you?    Out on my back when I was a small boy.
-Out on my back when I was a small boy.    Can I see your back?
-I'm abused.  Don't you think?    I don't think it's abuse, I think it's torture.
-...so we kill someone famous and if we are caught, we are sent to mental hospital...    Officers, there's your killer, do your duty, arrest him!
-...my little sister and I shared a flat - I came home one night and a man was raping her.  His gun was on the chair... He came at me and I shot him.    Alright.  That's a justifiable homicide.
-Alright.  That's a justifiable homicide.    Yes, but he was a cop.
-Now I become custody of police department?    If you cooperate with the DA - maybe they'll help you with your situation.
-If you cooperate with the DA - maybe they'll help you with your situation.    I will if they don't send me back.
-I will if they don't send me back.
```
Comment (tkornuta-nvidia) [inline tests/data/dialog_sample.txt]: Ok, so what actually happened with that file....? Something with spacing/tabs?

File: tests/test_tutorials_pytorch.py [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
+from tests.common_setup import NeMoUnitTest
+
+
+class TestPytorchChatBotTutorial(NeMoUnitTest):
+    def test_simple_train(self):
+        datafile = "tests/data/dialog_sample.txt"
+        logging.info(datafile)
+        voc, pairs = loadPrepareData("cornell", datafile=datafile)
+        self.assertEqual(voc.name, 'cornell')
+        self.assertEqual(voc.num_words, 675)
```
Comment (tkornuta-nvidia) [inline tests/test_tutorials_pytorch.py]: this looks good! well scoped, simple asserts, nice!

File: tests/data/dialog_sample.txt (hunk 1 above)
Comment (yzhang123) [reply tests/data/dialog_sample.txt]: yep, the tabs went missing, i added them back

**Pull request merged**